src/components/UriViewer.js

```diff
@@ -80,13 +80,13 @@ const PreviewContent = ({ uri, metadata, metadata: { bucket, name }, googleProje
     }
   }
   useOnMount(() => {
-    if (isGs(uri) && isFilePreviewable(metadata)) {
+    if (isFilePreviewable(metadata)) {
       loadPreview()
     }
   })
```

Comment:
We could save a good amount of vertical space with

```javascript
  useOnMount(() => isFilePreviewable(metadata) && loadPreview())
```

`someFlag && doSomething()` is also the pattern we usually follow for simple conditional statements.

Reply:
good call; adopted. I hadn't looked at refactoring the existing code like this, I was just making the minimal changes necessary.

Reply:
very understandable - thank you!